docs/faq.md

```diff
@@ -14,3 +14,7 @@
 - **How often do you release?** We strive to release the Splunk distribution within 2 working days after
   the [upstream project](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases)
   releases. OpenTelemetry Java generally releases a new version every 4 weeks.
+- **What happens when the exporter endpoint is temporarily unreachable?** Sending telemetry resumes when the endpoint
+  becomes available again. Telemetry collected in the meantime is dropped. To avoid dropping data, use the
```

Comment:
The `BatchSpanProcessor` will buffer spans for a little bit, so maybe this is a bit vague. Not sure that it matters to capture much more detail, but it's not completely accurate, right?